sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -397,16 +397,20 @@ function toRecognizedReceipt(result: DocumentResultModel, pages: FormPage[]): Re
   const form = toRecognizedForm(result, pages);
   return {
     recognizedForm: form,
-    locale: undefined
+    locale: undefined // in the future service would return locale info
   };
 }
 
 function toReceiptType(type: FormField): USReceiptType {
-  if (type.valueType === "string" && type.value === "Itemized") {
-    return "itemized";
-  } else {
-    return "unrecognized";
+  if (type.valueType === "string") {
```

Comment:
I get a little bit spooked out by this block. The type signature suggests that `"gas"` and `"parking"` are also valid values of `type`, and it feels like we're doing too much to transpose `FormField` into `USReceiptType` here. Is there any reason we couldn't return `type.valueType` in all cases where it's defined and then use `"unrecognized"` otherwise?

Reply:
I missed the `confidence` field on receipt type, it will be included in another PR so we will be returning `{ confidence: 1, type: "itemized" }

Reply:
oh it's in this PR

Reply:
So the `valueType` for a ReceiptType field should be "string" and the value of the field `type.value` is the receipt category, which is capitalized and I want to transform it into the camelCase that we normally use in TypeScript.

Reply:
If we are fine with capitalized string literals for receipt types then yes we can return them directly.

Reply:
hmm returning `type.value` which has a `string` type as stronger receipt type might not be valid here.

Reply:
Looks like TypeScript compiler can correctly narrow down the to the valid string literals.  I refactored this to use a switch statement. Hopefully it reads better.